rpm/stackdriver-agent.spec

```diff
@@ -365,6 +367,9 @@ fi
 %{_initddir}/stackdriver-agent
 %config(noreplace) /etc/sysconfig/stackdriver
 
+# Backward compatibility.
```

Comment:
Can you add instructions here and in stack-config about when the stack-config file and these two lines can be deleted (if ever)? Thanks.

Reply:
It may also be worthwhile to add comments in the other files for the stack-config lines.